.github/workflows/test_fuzzer_benchmarks.py

```diff
@@ -100,17 +105,30 @@ def do_build(build_type, fuzzer):
     else:
         raise Exception('Invalid build_type: %s' % build_type)
 
+    if always_build:
+        return make_builds(benchmarks, fuzzer)
+
+    changed_files = diff_utils.get_changed_files()
+    changed_fuzzers = change_utils.get_changed_fuzzers(changed_files)
+    if fuzzer in changed_fuzzers:
+        return make_builds(benchmarks, fuzzer)
+
```

Comment:
Maybe add comments for the three branches.

# Always build with all benchmarks if always_build is set.
# Build changed fuzzer with all benchmarks.
# Build fuzzer with changed benchmarks, skip rest.

Reply:
Done